.eslintrc.json

```diff
@@ -0,0 +1,60 @@
+{
+  "env": { "browser": true, "es2020": true },
```

Comment:
Why not es2021?

Reply:
Since ES2021 isn't finished yet, it's not a stable target; something could be added to it that Chrome hasn't implemented yet. This section is intended to list what assumptions we're allowed to make about the environment the code is running in (in this case, Chrome); I don't think features that TypeScript can downlevel are necessarily in scope.